test_autograph.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
```

Comment:
Follow up: Based on the kind of data processed in this test, I think it should be a python one. This would help to have it running automatically by tox. If we don't want people to run it by default, we can exclude with https://github.com/mozilla-releng/mozapkpublisher/blob/9853405889e03a1a75443d91302a5203170fabe6/mozapkpublisher/test/integration/test_get_l10n_strings.py#L12. 

We should probably create it under `signingscript/test/integration/test_autograph.py`. 

I can help to port it.

Reply:
Awesome thanks! Can you CC me on that bug?

It'll definitely be easier to verify the mar files from python with build mar.

For APK verification we've just been calling apksigner: https://github.com/mozilla-services/autograph-edge/blob/1f7733140b01eb70b2ffe58e454d60d5e4d5acbc/.circleci/config.yml#L88-L93

Reply:
Sure. PR is https://github.com/mozilla-releng/signingscript/pull/61. It's still WIP.

`apksigner` sounds good! For the record, we've used the old jarsigner in https://github.com/mozilla-releng/pushapkscript/blob/bc2d1c8eef6fffdf0f4b55be54f636c2feedd3bc/pushapkscript/jarsigner.py#L22 so far. There's also [androguard](https://pypi.python.org/pypi/androguard/) but it comes with a lot of python deps.

Reply:
Note to self: `apksigner` is part of Android build tools >= 24 https://developer.android.com/studio/releases/build-tools. On Ubuntu Xenial, only 23 is available https://packages.ubuntu.com/xenial/android-sdk-build-tools. Therefore, installing `apksigner` on Travis may be trickier than `jarsigner`.